test/repository_test.rb

```diff
@@ -163,6 +162,31 @@
         it 'raises an error when not persisted' do
           -> { UserRepository.update(user2) }.must_raise(Lotus::Model::NonPersistedEntityError)
         end
+
+        describe 'when entity contains updated_at without created_at' do
+          before do
+            CommentRepository.adapter    = adapter.new(mapper, uri)
+            CommentRepository.collection = :comments
+            @comment = CommentRepository.create(Comment.new())
+            @comment.instance_eval do
+              def created_at
+                @created_at
+              end
+            end
+          end
+
+          after do
+            CommentRepository.clear
+          end
+
+          it 'does not touch created_at' do
+            @comment.created_at.must_be_nil
+          end
+
+          it 'touches updated_at' do
+            @comment.updated_at.wont_be_nil
```

Comment:
we want to ensure that the new value is newer than before, can you please assert that here?